how-to/customize-workspace/client/src/modules/auth/example/util.ts

```diff
@@ -0,0 +1,24 @@
+import type { ExampleUser } from "./shapes";
+
+export function getCurrentUser(userSessionId: string): ExampleUser {
```

Comment:
Update signature to return `ExampleUser | null` to reinforce what the other return types are.

Reply:
dropped the key altogether and use platform uuid to enforce uniqueness across manifests.

Reply:
added null option